src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/TouchBehavior.android.cs

```diff
@@ -97,7 +97,7 @@ protected override void OnDetachedFrom(VisualElement bindable, AView platformVie
 		catch (ObjectDisposedException)
 		{
 			// Suppress exception
-			Trace.WriteLine("TouchBehavior is already disposed.");
+			Trace.TraceError("TouchBehavior is already disposed.");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.